4.6/alpine/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM alpine:3.4
+
+ENV NPM_CONFIG_LOGLEVEL info
+ENV NODE_VERSION 4.6.1
+
+RUN addgroup -S node \
+    && adduser -D -S -h /var/cache/node -s /sbin/nologin -G node node \
+    && apk add --no-cache \
+        libstdc++ \
+    && apk add --no-cache --virtual .build-deps \
+        binutils-gold \
+        curl \
+        g++ \
+        gcc \
+        gnupg \
+        libgcc \
+        linux-headers \
+        make -j$(getconf _NPROCESSORS_ONLN) \
```

Comment:
I guess that the -j option should be put on the make command (L40) rather than in the apk one